other/gaming.md

```diff
@@ -0,0 +1,15 @@
+# Cere Gaming
```

Comment:
We have gaming page in DDC v2 testnet. Don't we need to add it here?

Reply:
good point

https://games.dev.cere.network/